README.md

```diff
@@ -36,12 +53,53 @@ strfmt does not support empty identifiers (i.e. `{}` or `{:<10}`. Integer
 identifiers will be read as str keys to the hashmap (i.e. `{1:<10}` will have
 key == "1")
 
-## Legacy
+## Using the Map trait for customizing the display<a name="map-trait"></a> 
```

Comment:
This section can be removed. It's in the API docs

Reply:
As a counterpoint, I think the documentation of the Map trait should probably be there in the README since it highlights on user-end customization which is very prominent in a lot of README files. 

However if you still think we should remove this from the documentation, I'll take it off. 

PS: One of the reasons I had pulled your crate into my project as over some of the other ones was because I was easily able to override the HashMap requirement and customize my trait to cover lazy implementations